src/coreclr/jit/CMakeLists.txt

```diff
@@ -9,6 +9,8 @@ include_directories("../inc")
 # See https://github.com/dotnet/runtime/issues/33541
 if (CMAKE_CXX_COMPILER_ID STREQUAL "GNU" AND CMAKE_CXX_COMPILER_VERSION VERSION_LESS 11.0)
   add_compile_options(-Wno-error)
+else()
+  add_compile_options($<$<COMPILE_LANGUAGE:C,CXX>:/WX->) # don't treat warnings as errors
```

Comment:
If this is temporary its fine, but I don't expect anyone would want this in the final PR.

Reply:
I have no idea how I touched this... using VS probably.
I reverted.